osquery/logger/plugins/kafka_producer.cpp

```diff
@@ -26,17 +23,44 @@
 namespace osquery {
 
 FLAG(string,
-     logger_kafka_brokers,
+     kafka_brokers,
      "localhost",
      "Bootstrap broker(s) as a comma-separated list of host or host:port "
      "(default port 9092)");
+FLAG_ALIAS(std::string, logger_kafka_brokers, kafka_brokers);
 
-FLAG(string, logger_kafka_topic, "", "Kafka topic to publish logs under");
+FLAG(string, kafka_topic, "osquery", "Kafka topic to publish logs under");
+FLAG_ALIAS(std::string, logger_kafka_topic, kafka_topic);
 
 FLAG(string,
-     logger_kafka_acks,
+     kafka_acks,
      "all",
      "The number of acknowledgments the leader has to receive (0, 1, 'all')");
+FLAG_ALIAS(std::string, logger_kafka_acks, kafka_acks);
+
+FLAG(string,
+     kafka_security_protocol,
```

Comment:
Can this be `kafka_enable_tls` or something similar? I'm trying to accommodate for two things, most flags use the "disable" or "enable" prefix to turn off/on optional features, and we should use "tls" if we have the choice between that and "ssl".

Reply:
@theopolis Thanks for your comments. I was modeling these configuration properties after:
https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md

Some one opened a issue about supporting SASL and other options. Possible values supported by librdkafka are: plaintext, ssl, sasl_plaintext, sasl_ssl. Until SASL support is built into librdkafka formula we won't be able to add these extra options. Hence I left this as string instead of boolean.

Reply:
I would have preferred to just take any kafka_* CLI option and pass it directly to librdkafka (after converting underscore to period). That way users can add whatever they want. But I didn't see a easy way to do this with gflags.

Reply:
Ok, carry on :)